gadget/start_gadgetfs_RaspiZeroW.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
I think sudo is unnecessary as we assume the user runs as root. Were you running umap2 itself as non-root user?